vms/avm/network/network.go

```diff
@@ -31,27 +31,30 @@ var (
 type Network struct {
 	*p2p.Network
 
+	log       logging.Logger
```

Comment:
Moved this up to match the platformvm Network struct: https://github.com/ava-labs/avalanchego/blob/master/vms/platformvm/network/network.go#L29-L42